packages/core/src/App/Containers/Layout/header/__tests__/dtrader-contract-details-header.spec.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import { Router } from 'react-router-dom';
+import { createMemoryHistory } from 'history';
+import DTraderContractDetailsHeader from '../dtrader-v2-contract-detail-header';
+import userEvent from '@testing-library/user-event';
+
+jest.mock('@deriv-com/quill-ui', () => ({
+    Text: () => <div>Contract Details</div>,
```

Comment:
Sorry for silly questing, but why do we mocked Text component?

Reply:
its probably throwing an error